src/FsCodec.SystemTextJson/JsonRecordConverter.fs

```diff
@@ -66,11 +66,13 @@ type JsonRecordConverter<'T> (options: JsonSerializerOptions) =
                     |> Array.tryHead
                     |> Option.map (fun attr -> attr :?> JsonConverterAttribute)
                     |> Option.bind (fun attr ->
-                        let baseConverter = attr.CreateConverter(f.PropertyType)
-
-                        if baseConverter |> isNull then
-                            failwithf "Field %s is decorated with a JsonConverter attribute, but it does not implement a CreateConverter method." f.Name
-
+                        let baseConverter =
```

Comment:
I inferred this makes sense based on a) reading the xmldoc for JsonConverter in STJ b) the tests passing c) reading the converter authoring guide a bit
Seeking a review as its entirely possible I'm missing lots of obvious things...

Reply:
If it was mandatory, surely it'd be abstract?
For me, it seems the docs and xmldocs suggest that they don't all need to be converter factories
I guess we'll leave this here so someone can like to it in a told-you-so comment when we discover its wrong!